src/Models/NonhydrostaticModels/boundary_mass_fluxes.jl

```diff
@@ -47,52 +47,52 @@ function initialize_boundary_mass_fluxes(velocities::NamedTuple)
 
     # Check west boundary (u velocity)
     if u_bcs.west isa OBC
-        west_flux = get_boundary_flux(u_bcs.west, west_average(velocities.u))
+        west_flux = get_boundary_mass_flux(u_bcs.west, west_average(velocities.u))
         boundary_fluxes = merge(boundary_fluxes, (west = west_flux,))
```

Comment:
```suggestion
        boundary_fluxes = merge(boundary_fluxes, (; west = west_flux))
```

this is the style we agreed to use for singleton NamedTuple (but we can change convention if you like trailing comma better)